### PR TITLE
Issue #15218: formatted input file for section 4.8.7 modifiers to follow google style guide rules

### PR DESCRIPTION
**Pull request by Zopsss**
#15218

### CODE REVIEW

File: src/it/resources/com/google/checkstyle/test/chapter4formatting/rule487modifiers/InputModifierOrder.java [hunk 1]
```diff
@@ -1,300 +1,289 @@
...
 
-strictfp abstract class InputModifierOrder
-// violation above ''abstract' modifier out of order with the JLS suggestions.'
-{
-    transient private String dontSaveMe;
-    // violation above ''private' modifier out of order with the JLS suggestions.'
+// violation below ''abstract' modifier out of order with the JLS suggestions.'
+strictfp abstract class InputModifierOrder {
+  transient private String dontSaveMe;
+  // violation above ''private' modifier out of order with the JLS suggestions.'
```
Comment (rdiachenko) [inline src/it/resources/com/google/checkstyle/test/chapter4formatting/rule487modifiers/InputModifierOrder.java:6]: Why did you move violation comment? It does not look like a part of formatting.

In general, I think we should better be consistent in cases with a single violation. It is more readable to have `violation below` comment instead of `violation above` for simple cases. Here you mix both, let's be consistent.

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule487modifiers/InputModifierOrder.java:6]: It might be due to fact that formatter wants `InputModifierOrder {` but comment is in middle .

Did formatter move a comment ?

Comment (Zopsss) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule487modifiers/InputModifierOrder.java:6]: > It might be due to fact that formatter wants InputModifierOrder { but comment is in middle .

Yes

> Did formatter move a comment ?

No, I moved the comment and corrected the `{` placement

Comment (romani) [reply src/it/resources/com/google/checkstyle/test/chapter4formatting/rule487modifiers/InputModifierOrder.java:6]: >It is more readable to have violation below comment instead of violation above for simple cases. Here you mix both, let's be consistent.

We already have mix, if we do  this now, it will be more changes that you/we might not like also.
We still did not make good rules on when use each variant of comment.

**Review by rdiachenko — APPROVED** [review]
lgtm
